debian/dist/Ubuntu/control

```diff
@@ -162,7 +163,8 @@ Description: MariaDB database client binaries
 
 Package: mariadb-server-core-10.1
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, libmariadbclient18 (>= ${binary:Version})
+Depends: ${shlibs:Depends}, ${misc:Depends}, libmariadbclient18 (>= ${binary:Version}),
+    libsystemd-daemon0
```

Comment:
Do we really need to add new dependency? Isn't it added automatically to shlibs:Depends?